src/layout/content/box/Box.tsx

```diff
@@ -9,6 +9,7 @@ export interface IBoxProps {
     className?: string;
     metrics: IBoxMetrics;
     colors: IBoxColors | IBoxColorsThunk<ITheme>;
+    borderStyle?: keyof ITheme["spacing"]["borderStyle"];
```

Comment:
Why the extra level of indirection via theme instead of using the CSS type for border-style?